api/src/main/java/gov/va/vro/api/requests/GeneratePdfRequest.java

```diff
@@ -33,4 +30,8 @@ public class GeneratePdfRequest {
   @NonNull
   @Schema(description = "Medical evidence supporting assessment")
   private AbdEvidence evidence;
+
+  @Nullable
+  @Schema(description = "PDF template to generate", example = "v1")
+  private String pdfTemplate;
```

Comment:
@anthonyvelazquez has a pr adding more data in here as well. I think query parameters is better for these. Either way this end point will not be even used in mas workflow (or that is at least what I am expecting).

Reply:
I was using this endpoint for testing because I can control the exact input data to the PDF generator.